pages/table/expandable-rows-test.page.tsx

```diff
@@ -269,17 +269,20 @@ function useTableData() {
 
   // Imitate server-side delay when items update.
   const memoItems = useEqualsMemo(collectionResult.items);
-  const [readyItems, setReadyItems] = useState(memoItems);
+  const [delayedItems, setDelayedItems] = useState(memoItems);
+  const readyItems = settings.manualServerMock ? memoItems : delayedItems;
```

Comment:
What is the difference between `settings.manualServerMock` and `settings.useServerMock`?

In the dev page, there only seems to be one related option ("Use server mock")